### DISCUSSION TITLE
generate new users without email?

### DISCUSSION THREAD

**Discussion by fipp** [WeblateOrg/weblate#7772] [Q&A] [answered]
Hi! At my company we don't have a smtp server set up, we only send emails via an external API.

I created some weblate users, but realize that you need the email sent to proceed.

How can I solve this? Is there a way of doing it manually without the need for emails?

**Comment by nijel** [comment, marked as answer]
Maybe you can use that API to send out mails from Weblate as well? See https://docs.djangoproject.com/en/4.0/topics/email/#email-backends

You can also use another authentication method than e-mail/password one in case it does not work for you, see https://docs.weblate.org/en/latest/admin/auth.html
